<launch>
    <node pkg="l22_aero_code" type="logger.py" name="l22_aero_logger_node" output="screen" clear_params="true">
        <remap from="image_raw" to="main_camera/image_raw_throttled"/>
        <remap from="camera_info" to="main_camera/camera_info"/>
    </node>
    
</launch>